app/javascript/packages/build-sass/index.js

```diff
@@ -2,7 +2,7 @@ import { basename, join } from 'node:path';
 import { createWriteStream } from 'node:fs';
 import { Readable } from 'node:stream';
 import { pipeline } from 'node:stream/promises';
-import sass from 'sass-embedded';
+import { compile as sassCompile } from 'sass-embedded';
```

Comment:
This change was to address a deprecation which was causing some output from the build command:

```
`import sass from 'sass'` is deprecated.
Please use `import * as sass from 'sass'` instead.
```

Reply:
The rename from `compile` to `sassCompile` is just for readability, not a collision, right?

Reply:
> The rename from `compile` to `sassCompile` is just for readability, not a collision, right?

Correct, since we use a couple different processing tools (Sass, LightningCSS), I wanted to avoid potential ambiguity of what "compile" refers to, and follow the convention I'd introduced on [the following line](https://github.com/18F/identity-idp/blob/a20f6fb5aa1a36994098fe7f8f04293111785a79/app/javascript/packages/build-sass/index.js#L6) of with `lightningTransform` ➡️ `toolOperation` ➡️ `sassCompile`.